src/spec/page-objects/billing-page.js

```diff
@@ -71,13 +68,22 @@ class BillingPage {
     return this._secCodeInput.sendKeys(secCode);
   }
   isCcIconVisaDisplayed() {
-    return this._ccIconVisa.isDisplayed();
+    return this.hasCreditCardActiveClass('visa');
   }
   isCcIconMastercardDisplayed() {
-    return this._ccIconMaster.isDisplayed();
+    return this.hasCreditCardActiveClass('mastercard');
   }
   isCcIconAmexDisplayed() {
-    return this._ccIconAmex.isDisplayed();
+    return this.hasCreditCardActiveClass('amex');
+  }
+  hasCreditCardActiveClass(name) {
+    var hasClass = $('.credit-card')
```

Comment:
Can you send this container into a prop on top of the page? "$('.credit-card')" instead of calling it in the method.